<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>561</width>
    <height>374</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>YouTube Video downloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
background-color: rgb(148, 148, 148);
}
Line {
background-color: rgb(222, 222, 222);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QLineEdit" name="path">
      <property name="styleSheet">
       <string notr="true">#path {
	border-width: 1.5px;
	border-style: solid;
	border-color: rgb(79, 79, 79);
	background-color: #52595d;
	color: white;
	border-radius: 80px;
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="pathButton">
      <property name="styleSheet">
       <string notr="true">#pathButton {
	padding: 2px;
	border: 2px solid rgb(85, 85, 85);
	color: rgb(50, 50, 50);
	background-color: white;
	font: 9pt &quot;Bahnschrift SemiBold&quot;;
}
#pathButton:hover {
	padding: 2px;
	color: rgb(255, 255, 255);
	border: none;
	background-color: rgb(85, 85, 85);
	font: 9pt &quot;Bahnschrift SemiBold&quot;;
	
}</string>
      </property>
      <property name="text">
       <string>Choose path</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="name">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="2">
     <widget class="QProgressBar" name="pBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#pBar {
	padding: 4px;
	text-align: center;
	color: #d3dae3;
	border-radius: 10px;
	border-color: transparent;
	border-style: solid;
	background-color: #52595d;
}
#pBar::chunk {
	background-color: #214037	;
	border-radius: 10px;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">#lineEdit {
	border-width: 1.5px;
	border-style: solid;
	border-color: rgb(79, 79, 79);
	background-color: #52595d;
	color: white;
	border-radius: 80px;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Video URL</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="downloadButton">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#downloadButton {
	padding: 2px;
	border: 2px solid rgb(85, 85, 85);
	color: rgb(50, 50, 50);
	background-color: white;
	font: 9pt &quot;Bahnschrift SemiBold&quot;;
}
#downloadButton:hover {
	padding: 2px;
	color: rgb(255, 255, 255);
	border: none;
	background-color: rgb(85, 85, 85);
	font: 9pt &quot;Bahnschrift SemiBold&quot;;
	
}</string>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="image">
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>270</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="4" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
